src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const list = document.querySelectorAll('.population');
+const total = [...list].reduce(
+  (acc, item) => acc + Number(item.innerText.replace(/,/g, '')),
+  0
+);
+
+const average = total / [...list].length;
+
+const totalString = total.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+const averageString = average.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+
+document.querySelector('.total-population').innerHTML = totalString;
+
+document.querySelector('.average-population').innerHTML = averageString;
```

Comment:
I think it's better to store these values (total-population and average-population) in separate variables at the beginning of the file
this way it's easy to follow the code

```suggestion
const total = document.querySelector('.total-population')

total.innerHTML = totalString;
```